DllImportGenerator/DllImportGenerator/Analyzers/ManualTypeMarshallingAnalyzer.cs

```diff
@@ -316,6 +330,12 @@ private void AnalyzeNativeMarshalerType(SymbolAnalysisContext context, ITypeSymb
                     return;
                 }
 
+                if (marshalerType.IsUnboundGenericType)
+                {
+                    context.ReportDiagnostic(Diagnostic.Create(NativeGenericTypeMustBeClosedRule, nativeMarshalerAttributeData.ApplicationSyntaxReference!.GetSyntax().GetLocation(), nativeType.ToDisplayString(), type.ToDisplayString()));
```

Comment:
Seems some of the other diagnostics were updated to have some arguments on separate lines but this one wasn't, any reason?